### ISSUE TITLE
Dataloader messes with numpy seed

### ISSUE THREAD

**Issue by Atcold** [Atcold/pytorch-PPUU#32]
https://github.com/Atcold/pytorch-PPUU/blob/master/dataloader.py#L91-L92
```python
# Inside an else-statement
numpy.random.seed(0)
perm = numpy.random.permutation(self.n_episodes)
```
should be instead
```python
rng = np.random.RandomState(0)
rng.permutation(self.n_episodes)
```
Otherwise `dataloader` will change stuff conditionally...
Need to verify consistency, though.
